app/models/plant_population.rb

```diff
@@ -87,9 +87,7 @@ def as_indexed_json(options = {})
   def self.permitted_params
     [
       :fetch,
-      query: [
-        :id, :name
-      ]
+      query: [:id, :name]
```

Comment:
In my PR I decided, mainly for clarity and consistency, to make all query parameters strings, not symbols (some include a dot, and keeping them all symbols would require ugly syntax..).

Will switching that to strings, not symbols, be of any problem to this PR?